concordia/admin/__init__.py

```diff
@@ -310,13 +310,13 @@ class ItemAdmin(admin.ModelAdmin):
 
     actions = (publish_item_action, unpublish_item_action)
 
-    def get_deleted_objects(self, objs, request):
+    def get_deleted_objects(self, objs, request):  
         deleted_objects = [str(obj) for obj in objs]
```

Comment:
There are two things to be concerned with here: Item does not use a related lookup in its default string representation but if it did this would trigger O(n) queries unless the `QuerySet` used `select_related` and/or `prefetch_related`. For bulk deletion, there could be some value in a heuristic which did something like using `str(obj.pk)` if the queryset length was over a certain value.

Related, this will generate one list item per object. If you are deleting hundreds or thousands of objects this is not always desirable and it might be preferable to have that heuristic do something like this:

```python
if len(objs) < 30:
    deleted_objects = [str(obj) for obj in objs]
else:
    deleted_objects = [str(obj) for obj in objs[:3], f"… and {len(objs) - 3} more {obj._meta.verbose_name_plural}"]
```

Reply:
Ok all the changes from above are completed and pushed except the one about the deleted_objects, not sure how to code this part - > deleted_objects = [str(obj) for obj in objs[:3], f"… and {len(objs) - 3} more {obj._meta.verbose_name_plural}"]